api/app/Http/Helper/Mail/helpers.php

```diff
@@ -0,0 +1,46 @@
+<?php
+ /*this is a common method for sending mails to users
+ data : users data
+ subject : whatyour mail subject should be
+ to_list : whom is the mail for
+ cc_list : list of people in cc list (optional)
+ bcc_list : list of people in bcc list (optional)
+ */
+
+function send_mails($data,$subject,$to_list,$cc_list=NULL,$bcc_list=NULL)
+{
+	
+	if (!empty($cc_list) && !empty($bcc_list)) 
```

Comment:
@jalmiShreya - use if else conditions to decide variable values. Email send method then could only be called once.